### PR TITLE
Make date columns sortable

### LINKED ISSUE #94: enhancing the sorting option of columns  for the date attributes
### What would you like to be added?

Hello

it will be great that we can sort the table by the sorting options like A to Z, that is already included; but it is not complete i think.
i think one of the most important and efficient type of sorting is sorting based on the increasing or decreasing date.
now if we have date attribute in the metadata of notes, the plugin can detect it, but the sorting option of this column is based on A to Z; it will be great to enhance this sorting option at least for data attribute to able to sorting the notes in the table form based on decreasing or increasing of date.

### Why is this needed?

it will be super helpful to see the notes in the table based on the  decreasing or increasing orientation of date that is inserted in the metadata of notes.

### PR DESCRIPTION
**Pull request by GamerGirlandCo**
closes #94

### CODE REVIEW

**Review by marcusolsson — CHANGES_REQUESTED** [review]
Thank you for contributing! 

Looks like there's some inconsistent formatting. You should be able to run the following to fix it:

```bash
yarn run prettier . --write
```

Otherwise, looks great! 💪

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # vscode
 .vscode
+.history/
```
Comment (marcusolsson) [inline .gitignore:3]: If this is helpful to you, I don't mind adding it, but what's it for? :)

File: tsconfig.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
     "noPropertyAccessFromIndexSignature": true,
     "allowUnreachableCode": false,
     "allowUnusedLabels": false,
+    "allowSyntheticDefaultImports": true,
```
Comment (marcusolsson) [inline tsconfig.json:20]: This doesn't seem like it's related to this pull request?

File: .gitignore (hunk 1 above)
Comment (GamerGirlandCo) [reply .gitignore:3]: Oh, that's just so that files created by the vscode file history plugin don't clutter up the changed files section!

**Review by marcusolsson — APPROVED** [review]

**Comment by marcusolsson** [comment]
Why is this PR updating the yarn.lock? Is there a dependency that you need?

**Comment by marcusolsson** [comment]
Since this PR doesn't need to update any dependencies. Could you remove it for now? 🙏

**Pull request merged**